### PR TITLE
Update readme for running commands

### PR DESCRIPTION
**Pull request by tai271828**
I was aware that some typos in the commands of README. This pull request fixes them and update the corresponding output result.

Bonus change: re-structure the heads of the document.

**PR made with love in PyCon TW 2020 sprint event!!**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,43 +64,127 @@ $ pipenv shell
...
 
 ```bash
-docker build . -t quark
+$ quark --help
 ```
-Examples:
+
+#### Test It Out
+
+Firstly let;s fetch `rules` and some apk examples.
```
Comment (18z) [inline README.md:75]: typo: let;s

File: README.md [hunk 2]
```diff
@@ -20,7 +20,13 @@ Malware evolved with new techniques to gain difficulties for reverse engineering
 
 Our Dalvik bytecode loader consists of functionalities such as 1. Finding cross reference and calling sequence of the native API. 2. Tracing the bytecode register. The combination of these functionalities (yes, the order theory) not only can neglect obfuscation but also match perfectly to the design of our malware scoring system.
 
-### Detail Report
+### Easy to Use and Reading Friendly Report
+
+Quark is very easy to use and also provides flexible output format. There are 3 types of output report: detail, call graph, and summary.
```
Comment (krnick) [inline README.md:25]: provides a flexible

Comment (tai271828) [reply README.md]: will be `flexible output formatS`

**Comment by krnick** [comment]
@tai271828 thanks for helping us optimize our documentation!

**Pull request merged**